### PR TITLE
feat!: remove any references to repository tiers

### PR DESCRIPTION
**Pull request by ctreatma**
In practice, repository tiers did not provide additional context on how or how well a given repository was maintained.  Given that we have not found any precedent for these tiers in the broader Open Source world, we are removing the tiers and relying on the license in each repository to establish what, if any, maintenance expectations are valid for that repository.

### CODE REVIEW

**Comment by cprivitere** [comment]
It seems like it would still be good to have some sort of marking that something is EOL or deprecated or replaced by something else. Do we just want to use github archiving to indicate that?

**Comment by displague** [comment]
As helpful as the badges and tiers _appear to be_, they do not accurately represent expectations and certainly across the dozens of projects that have referred to these tiers.

Most of these projects include the [MIT](https://opensource.org/license/MIT) or [Apache 2](https://opensource.org/license/apache-2-0#:~:text=Disclaimer%20of%20Warranty.) license. Both of these licenses, as well as the others that may appear, more effectively define expectations for these projects.

In cases where a project has already been deprecated and archived, references to tier documentation may 404. This is unfortunate, but we can't expect archived project links to stay evergreen.

In projects that will be deprecated and archived, we can use a short README.md update stating as much and linking to the new project. This is a common and informal practice in repositories.

In projects that refer to these tiers, we'll want to open issues (or PRs) to remove those tiers, referring back to this PR.

**Review by displague — APPROVED** [review]

**Comment by ctreatma** [comment]
> It seems like it would still be good to have some sort of marking that something is EOL or deprecated or replaced by something else. Do we just want to use github archiving to indicate that?

We talked about archiving repos to indicate EOL and maybe deprecation; IMO that's a good practice but we don't really have a way (or even a desire, I think) to enforce that.  A maintainer could also document elsewhere (e.g. in their repo's README) that the project is deprecated or end-of-life, but that's their call and we're no longer providing the standard, centralized text for those sorts of notices.

**Pull request merged**
